src/documentation-generator.js

```diff
@@ -9,37 +9,6 @@ function removeDocumentationFiles() {
   rimraf.sync(outputDir);
 }
 
-function isExternal(type) {
```

Comment:
Removing these custom filters because TypeDoc handles this automatically now, with the `stripInternal` setting (see below).